### PR TITLE
Updated Frame Settings with new names and added fields

### PR DESCRIPTION
**Pull request by JordanL8**
### Purpose of this PR
Updated the Frame Settings doc with missing fields and also changed names to match those in Remi's PR (https://github.com/Unity-Technologies/ScriptableRenderPipeline/pull/4366). Also small fix in the MSAA doc.

---
### Overall Product Risks

**Technical Risk**: None
**Halo Effect**: None

---
### Comments to reviewers
@RSlysz can you double check everything is in the right order/has the right name?

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md [hunk 1]
```diff
@@ -25,21 +25,37 @@ These settings determine the method that the Cameras and Reflection Probes using
...
 | **Transparent Prepass**           | Enable the checkbox to make HDRP perform a Transparent Prepass. Enabling this feature causes HDRP to add polygons from transparent Materials to the depth buffer to improve sorting. |
-| **Transparent Postpass**          | Enable the checkbox to make HDRP perform a Transparent Postpass. Enabling this feature causes HDRP to add polygons to the depth buffer that postprocessing uses. |
-| **Transparents Write Velocity**   | Enable the checkbox to allow HDRP to write the velocity of transparent GameObjects to the velocity buffer. To make HDRP write transparent GameObjects to the velocity buffer, you must also enable the **Transparent Writes Velocity** checkbox on each transparent Material. Enabling this feature means that effects, such as motion blur, affect transparent GameObjects.  This is useful for alpha blended objects like hair. |
+| **Low Resolution Transparent**    | Enable the checkbox to allow HDRP to perform a low resolution render pass. If you disable this checkbox, HDRP renders transparent Materials using the **Low Resolution** render pass in full resolution. |
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md]: Oops. I made a mistake here in my PR. Low Resolution Transparent should be after Transparent PostPass. I will fix it in my PR too.

File: com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md [hunk 2]
```diff
@@ -49,39 +65,43 @@ These settings determine the method that the Cameras and Reflection Probes using
...
+| **Transmission**                   | Enable the checkbox to make HDRP process the transmission effect. This allows subsurface scattering Materials to use transmission, for example, light transmits through a leaf with a subsurface scattering Material. |
+| **Fog**                            | Enable the checkbox to make HDRP process atmospheric scattering. This allows your Camera/Reflection Probe to process atmospheric scattering effects such as the [fog](HDRP-Features.html#FogOverview.html) from your Scene’s Volumes. |
+| **Volumetrics**                    | Enable the checkbox to make HDRP process Volumetrics. Enabling this setting allows your rendering component to render volumetric fog and lighting. |
+| **Reprojection For Volumetrics**   | Enable the checkbox to improve the quality of volumetrics at runtime. Enabling this feature causes HDRP to use several previous frames to calculate the volumetric effects. Using these previous frames helps to reduce noise and smooth out the effects. |
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md]: Also, just changed Reprojection For Volumetrics to just Reprojection as it is indented under it.

Can you indent title as in the editor for better clarity ?

File: com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md (hunk 1 above)
Comment (JordanL8) [reply com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md]: Fixed it

**Comment by RSlysz** [comment]
@JulienIgnace-Unity for merging

**Pull request merged**